week-2/L-array-methods-2/exercise2.js

```diff
@@ -7,7 +7,9 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  return str.split
```

Comment:
This doesn't seem like it would work :-) What are you thinking about here? How can we help?